clients/drcachesim/tracer/instr_counter.cpp

```diff
@@ -470,6 +473,13 @@ parse_instr_intervals_file(std::string path_to_file)
         if (!std::getline(ss, elem, ','))
             FATAL("Fatal error: instruction duration not found.\n");
         uint64 duration = std::stoull(elem);
+        if (duration == 0) {
+            NOTIFY(0,
+                   "Instruction interval starting at %" PRIu64 " has duration of 0. "
+                   "Removing interval.\n",
+                   start);
+            continue;
```

Comment:
The main changes from the last review are:
1) Using `UIN64_MAX` instead of `0` for the last "no trace" window we add.
2) Added a `NOTIFY` message if the user-defined instruction intervals contain intervals of duration == 0 (we skip it).
3) Added a `NOTIFY` message when we merge overlapping intervals.